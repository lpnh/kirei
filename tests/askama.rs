mod helper;

#[test]
fn comment() {
    format_and_snapshot!("askama/comment.html");
}

#[test]
fn nested_comment() {
    format_and_snapshot!("askama/nested_comment.html");
}

#[test]
fn if_block() {
    format_and_snapshot!("askama/if_block.html");
}

#[test]
fn let_block() {
    format_and_snapshot!("askama/let_block.html");
}

#[test]
fn filter() {
    format_and_snapshot!("askama/filter.html");
}
