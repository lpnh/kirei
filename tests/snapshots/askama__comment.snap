---
source: tests/askama.rs
description: "{# A Comment #}\n"
---
{# A Comment #}
